### PR TITLE
Update home_controller.rb

### PR DESCRIPTION
**Pull request by mikicaivosevic**
Refactor toggle_mobile

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -26,13 +26,7 @@ def show
...
 
   def toggle_mobile
-    if session[:mobile_view].nil?
-      # we're most probably not on mobile, but user wants it anyway
-      session[:mobile_view] = true
-    else
-      # switch from mobile to normal html
-      session[:mobile_view] = !session[:mobile_view]
-    end
+    session[:mobile_view] = session[:mobile_view].nil ? true : !session[:mobile_view]
```
Comment (jhass) [inline app/controllers/home_controller.rb]: Typo here, `.nil` -> `.nil?`. You can catch these easily by [running tests locally](https://wiki.diasporafoundation.org/Testing_workflow).

**Comment by jhass** [comment]
Ping

**Comment by mikicaivosevic** [comment]
It's ok now. :)

**Comment by jhass** [comment]
Merged as 78b0fbbee64555b76eee73a0fd46c7b5ef7bf062

Thanks!

**Pull request closed without merging**